blackbox/handlers/notifiers/discord.py

```diff
@@ -39,6 +39,10 @@ def _parse_report(self, report: dict) -> dict:
                 emoji = ":white_check_mark:" if provider['success'] else ":x:"
                 field['value'] += f"{emoji}  {provider['type']}\n"
 
+            # When backup failed, add X emoji to value to avoid error.
```

Comment:
Which error are we avoiding? is this really happening "when backup failed"? This comment could be improved.